src/components/ListView/index.jsx

```diff
@@ -1,30 +1,21 @@
 import React from 'react';
 import { useCollection } from 'react-firebase-hooks/firestore';
 import { getProducts } from 'components/Utils/firestore';
+import ProductList from 'components/ProductList';
 
 function ListView() {
   const query = getProducts();
   const [values, loading, error] = useCollection(query);
 
   return (
-    <div>
-      <h2>List</h2>
+    <>
+      <h2>Smart Shopping List</h2>
       <div>
         {error && <strong>Error: {JSON.stringify(error)}</strong>}
         {loading && <span>List: Loading...</span>}
```

Comment:
I left a comment about this in the other pair's PR: https://github.com/the-collab-lab/tcl-18-smart-shopping-list/pull/20/files#r567335194

Let's let them change it but I wanted y'all to get the learning as well

Reply:
Done.